### PR TITLE
fix(chore): Dockerfile did remove the `ROOT` folder belonging to liferay

### PR DESCRIPTION
**Pull request by maxhbr**
This belongs to a future feature and creates a fatal bug.

### CODE REVIEW

**Review by lepokle — APPROVED** [review]

File: deployment/sw360/Dockerfile [hunk 1]
```diff
@@ -12,7 +12,6 @@ MAINTAINER admin@sw360.org
 
 RUN apk add --update bash openssl fontconfig
 
-ADD sw360_dependencies.tar.gz /opt
```
Comment (alexbrdn) [inline deployment/sw360/Dockerfile:15]: I get it with the ROOT folder, but why this change?

Comment (lepokle) [reply deployment/sw360/Dockerfile:15]: Because this line was a duplicate

Comment (lepokle) [reply deployment/sw360/Dockerfile:15]: to line 21

Comment (alexbrdn) [reply deployment/sw360/Dockerfile:15]: Thanks
